src/main/webapp/WEB-INF/tags/instructor/results/remindButton.tag

```diff
@@ -0,0 +1,13 @@
+<%@ tag description="instructorFeedbackResults - remind button" %>
+<%@ tag import="teammates.common.util.Const" %>
+
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+
+<%@ attribute name="remindButton" type="teammates.ui.template.InstructorFeedbackResultsRemindButton" required="true" %>
+
+<div style="display: inline-block; padding-right: 5px;" class="remind-no-response">
+    <a href="#" data-actionlink="${remindButton.urlLink}" class="${remindButton.className}" data-toggle="modal"
+            <c:if test="${remindButton.disabled}">disabled="disabled"</c:if> data-target="#remindModal">
```

Comment:
@shubham49 yes, please do.

@damithc In theory, `disabled="disabled"` could be better because it counts as valid XHTML, while the other variations do not. However, this point is rendered moot as our standards also forbid the use of closing tags for void elements, which is mandatory in XHTML.

In terms of jQuery, setting an element's `disabled` property to `true` using `.prop()` causes `disabled=""` to be added to the element in the DOM. This applies to all other possible properties that can be added.

XHTML and jQuery aside, all three variations are equivalent in HTML. Therefore, I would suggest the use of `disabled=""` to be consistent with the behaviour of jQuery.